data/debug-cli/docker_debug.yaml

```diff
@@ -0,0 +1,166 @@
+command: docker debug
+short: Get a shell into any container or image.
+long: |-
+  > **Note**
+  >
+  > Docker Debug requires a [Pro, Team, or Business subcription](/subscription/details/).
+  > You must [sign in](/desktop/get-started/) to use this command.
+
+  Get an enhanced shell with additional tools into any container or image.
+  
+  The shell contains the following builtin commands:
+  - `install [tool1] [tool2]`: Add Nix packages from: https://search.nixos.org/packages.
```

Comment:
We should add a note here that installed tools never get added to the actual container/image, just to your debug toolbox. This is very different behaviour people are used to from eg `docker exec`. People ask for clarification on this a lot.

Reply:
Added some more info below that paragraph and in the examples.